models/classes/session/Business/Service/SessionCookieService.php

```diff
@@ -68,15 +69,12 @@ public function initializeSessionCookie(): void
             //cookie keep alive, if lifetime is not 0
             if ($sessionParams['lifetime'] !== 0) {
                 $expiryTime = $sessionParams['lifetime'] + time();
-                setcookie(
-                    GENERIS_SESSION_NAME,
-                    session_id(),
-                    $expiryTime,
-                    (string)$sessionCookieAttributes,
-                    $cookieDomain,
-                    $isSecureFlag,
-                    true
-                );
+                setcookie(GENERIS_SESSION_NAME, session_id(), [
+                    'expires' => $expiryTime,
+                    'domain' => $cookieDomain,
+                    'secure' => $isSecureFlag,
+                    'httponly' => true,
```

Comment:
use value from `$cookieParams`
```suggestion
                    'httponly' => $cookieParams['httponly'],
```